src/custom/utils/format.ts

```diff
@@ -32,13 +50,15 @@ export function formatSmart(
 
   const precision = value instanceof CurrencyAmount ? value.currency.decimals : 0
   const amount = value instanceof CurrencyAmount ? value.quotient.toString() : value.toFixed(decimalsToShow)
+  // Take the min(decimalsToShow, token decimals) because a token can have less decimals than the what we want to show
+  const smallLimitPrecision = Math.min(decimalsToShow, precision || DEFAULT_DECIMALS)
```

Comment:
precision here is always a value - tho this ternary will see the `0` fallthrough condition in the ternary as false and take DEFAULT_DECIMALS. This is intended? else do 
`const smallLimitPrecision = Math.min(decimalsToShow, precision ?? DEFAULT_DECIMALS)`

Reply:
This is a tricky one.

I'll do as you suggest because I don't remember the reason why I did it like that... which I knew when I started typing but lost track :shrug: